### PR TITLE
Remove usused direct dependencies on 'webpack'

### PR DESCRIPTION
**Pull request by trivikr**
Fixes: https://github.com/bluesky-social/social-app/issues/6461

Ensured that `yarn web` runs with a clean build, and website is accessible at http://localhost:19006/

```console
$ git clean -dfx && yarn

$ yarn web
yarn run v1.22.19
$ expo start --web
Starting project at /Users/trivikram/workspace/social-app
Starting Metro Bundler
Starting Webpack on port 19006 in development mode.
...
```

Verified that there's no `babel-loader` folder in `node_modules/.cache`
```console
$ ls node_modules/.cache      
web
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -221,7 +220,6 @@
     "@react-native/typescript-config": "^0.74.1",
     "@testing-library/jest-native": "^5.4.1",
     "@testing-library/react-native": "^11.5.2",
-    "@tsconfig/react-native": "^2.0.3",
```
Comment (gaearon) [inline package.json:224]: what was this for?

**Comment by gaearon** [comment]
This would need a rebase.

File: package.json (hunk 1 above)
Comment (trivikr) [reply package.json:224]: The tsconfig.json was extended from `@tsconfig/react-native` in https://github.com/bluesky-social/social-app/commit/d6ec627c8cd32836e5ed494606318959ca17fca1#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0

It was changed to `@react-native/typescript-config` in https://github.com/bluesky-social/social-app/commit/93f5bb3b1f10cb6011f1770132067fc58df39f3b#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R2

**Comment by trivikr** [comment]
Rebase done.

**Review by gaearon — APPROVED** [review]
lgtm, thank you

**Pull request merged**